### PR TITLE
Frontend overhaul

### PR DESCRIPTION
**Pull request by Primdahl26**
This does quite a lot of things with our UI...

First of all everything but the `multi-select` is ready for themes to be implemented.

This PR does:
1. Change the index page to look more slick with removing the streamchaser text that was there for no reason. Adds shadows, a neutral colour under the image, and a new on hover effect.
image

image

2. Has the same slick to detailsview, and the same on hover effect has been applied to clickable items. Providers has been redone so it looks a lot cleaner.

image

image

3. Things have been made to use `daisyui` colors more correctly, so that primary is indicated when we want someone to press it. neutral is being used when its a background for a div, or a button like `Show less`. This works on all the `daisyui` themes i tested.

### CODE REVIEW

**Comment by AndreasPB** [comment]
Add some screenshots please :D

**Comment by Primdahl26** [comment]
image

File: frontend/src/components/media_search.svelte [hunk 1]
```diff
@@ -27,15 +27,18 @@
 <!-- TODO: Why is both checks needed? -->
 {#if media.hits}
     {#if media.hits.length}
-        <div class="grid grid-cols-2 2xl:grid-cols-7 xl:grid-cols-6 lg:grid-cols-5 md:grid-cols-4 sm:grid-cols-3 gap-2 px-4 pt-2 pb-4 bg-base-100">
+        <div class="grid grid-cols-2 2xl:grid-cols-7 xl:grid-cols-6 lg:grid-cols-5
+                    md:grid-cols-4 sm:grid-cols-3 gap-2 px-4 pt-2 pb-4">
             {#each media.hits as media, mediaIndex}
                 <a href="{mediaIdToUrlConverter(media.id)}"
-                    class="card compact bordered w-auto transition duration-500 ease-in-out cursor-pointer transform hover:scale-110 m-1">
+                    class="card compact w-auto transition duration-500 ease-in-out
```
Comment (AndreasPB) [inline frontend/src/components/media_search.svelte:34]: The 500ms duration makes it feels really sluggish, not sure if we should remove it or turn it down to something like 100

**Review by AndreasPB — APPROVED** [review]
Sick improvements :ship:

**Pull request merged**